scripts/populate_preprint_providers.py

```diff
@@ -60,7 +59,13 @@ def set_or_update_m2m_fields(provider, emails, licenses, links, social_accounts)
         provider.licenses_acceptable.add(*licenses)
 
     for link in links:
-        obj, created = PreprintProviderLink.objects.get_or_create(**link)
+        obj, created = provider.links.update_or_create(
```

Comment:
Actually, no. It will put `'FACESPACE IS THE BEST'` in linked_text if it's empty. It'll be thrown away otherwise.